src/Geopilot.Frontend/src/pages/delivery/deliverySubmit.tsx

```diff
@@ -68,6 +55,16 @@ export const DeliverySubmit = () => {
               values={mandates
                 ?.sort((a, b) => a.name.localeCompare(b.name))
                 .map(mandate => ({ key: mandate.id, name: mandate.name }))}
+              onUpdate={mandateId => {
+                formMethods.setValue("predecessor", undefined);
+                if (mandateId) {
+                  fetchApi<Delivery[]>(
+                    "/api/v1/delivery?" + new URLSearchParams({ mandateId: mandateId.toString() }),
+                  ).then(setPreviousDeliveries);
+                } else {
+                  setPreviousDeliveries([]);
+                }
+              }}
```

Comment:
PP: Funktion in der Komponente ausserhalb vom JSX definieren & hier nur referenzieren.